### PR TITLE
Add ingress for zooniverse.org

### PR DESCRIPTION
**Pull request by adammcmaster**
In preparation for switching frontend sites over to Azure, I'm going to start adding *.zooniverse.org domains to an ingress here. I'm starting with the bare Zooniverse.org domain, and in another PR I plan to add more records which are currently being served directly by the static ELB.

### CODE REVIEW

File: kubernetes/ingress/zooniverse.org.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    - zooniverse.org
+    secretName: zooniverse-org-tls-secret
+  rules:
+  - host: zooniverse.org
+    http:
+      paths:
+      - backend:
+          serviceName: http-frontend
+          servicePort: 80
+        path: /(.*)
```
Comment (camallen) [inline kubernetes/ingress/zooniverse.org.yaml:23]: OOI i assume if we don't specify the path regex here and use that in the `rewrite-target` we lose the path matching on the incoming requests?

Basically is this is needed if we're matching all paths and just passing to the upstream as is...?

File: kubernetes/ingress/zooniverse.org.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    kubernetes.io/ingress.class: nginx
+    certmanager.k8s.io/cluster-issuer: letsencrypt-prod
+    nginx.ingress.kubernetes.io/rewrite-target: /$1
+    nginx.ingress.kubernetes.io/proxy-buffer-size: "128k"
+    nginx.ingress.kubernetes.io/set-real-ip-from: "10.0.0.0/8"
+spec:
+  tls:
+  - hosts:
+    - zooniverse.org
+    secretName: zooniverse-org-tls-secret
```
Comment (camallen) [inline kubernetes/ingress/zooniverse.org.yaml:15]: And this secret will be autogenerated by the cluste cert issuer (letsencrypt) on deploy, right?

Comment (adammcmaster) [reply kubernetes/ingress/zooniverse.org.yaml:15]: Yup

File: kubernetes/ingress/zooniverse.org.yaml (hunk 1 above)
Comment (adammcmaster) [reply kubernetes/ingress/zooniverse.org.yaml:23]: I'm pretty sure it's needed if I've understood the docs correctly.

**Comment by adammcmaster** [comment]
> OOI do we have traffic on zooniverse.org now? And if not what's the plan for this domain long term? Redirection to `www` subdomain?
> 
> ```
> $ curl -v zooniverse.org
> * Rebuilt URL to: zooniverse.org/
> * Could not resolve host: zooniverse.org
> * Closing connection 0
> curl: (6) Could not resolve host: zooniverse.org
> ```

Looks like I accidentally broke this yesterday when I was making some changes to the DNS (removing ALIAS records in preparation for moving to Azure DNS). It's set up to redirect to www already.

**Pull request merged**